grades/models.py

```diff
@@ -79,3 +90,60 @@ def __str__(self):
             user=self.final_grade.user,
             course_id=self.final_grade.course_run.edx_course_key
         )
+
+
+class FinalGradeRunInfo(TimestampedModel):
```

Comment:
i wonder if `CourseRunGradeStatus` or `CourseRunGradingStatus` would be a better name here. `Info` is really ambiguous and it doesn't help convey the purpose of this model. compare the following: `FinalGradeRunInfo. is_course_run_complete(course_run)` vs. `CourseRunGradeStatus.is_complete(course_run)`. the second one is much more helpful and readable, IMO